include/sfx.h

```diff
@@ -426,6 +414,49 @@ enum Sfx {
     SFX_DEATH_AMBIENCE,    // BGM ambience during first Death cutscene
     SFX_MAGIC_GLASS_BREAK, // Subweapon Container
     SFX_UI_CONFIRM,
+    SFX_CANDLE_HIT,
+    SFX_TELEPORT_BANG_A, // Used when teleport starts
+    SFX_TELEPORT_BANG_B, // Used when teleport ends, but is also Warp Enter
+    SFX_SUC_APPEAR,      // Succubus reveal
+    SFX_UNK_SCRAPE_A,
+    SFX_UNK_SCRAPE_B,
+    SFX_UNK_SCRAPE_C,
+    SFX_UNK_UI_SELECT,
+    SFX_START_SLAM_A, // Dark Shield
+    SFX_START_SLAM_B, // "Press Start", Door Open, Dracula Stomp, etc.
```

Comment:
Having just gone through this list to find a particular symbol, does it make sense to add a value marker comment every 0x10 entries or so?

```c
    ⋮
    // 0x630
    SFX_UNK_BETA_630, // MAD, TE1, …
    ⋮
    // 0x640
    SFX_ANIME_SWORD_B,
    ⋮
```

Reply:
I think this is fine but I'd put it to the right so it doesn't conflict with the vab info 
```SFX_ANIME_SWORD_B, // 0x640```

Reply:
That's a great idea, I'll be sure to do that.

Reply:
I'll be sure to add this in the next commit.